### ISSUE TITLE
Error when accessing `flask.request.data`

### ISSUE THREAD

**Issue by adamtheturtle** [adamtheturtle/mypy-strict-kwargs#26]
Current most minimal example:

```python
from typing import Any


class decorator(property):
    def __get__(self, obj: object, objtype: type | None = None) -> Any:
        return


class C:
    @decorator
    def prop(self) -> None:
        pass


c = C()
c.prop
```

**Comment by adamtheturtle** [comment]
Setter example:

```python
from typing import Any


class decorator(property):
    def __get__(self, obj: object, objtype: type | None = None) -> Any: ...

    def __set__(self, instance: object, value: Any) -> None: ...


class C:
    @decorator
    def prop(self) -> str:
        return self._val

    @prop.setter
    def prop_set(self, new_value: str) -> None:
        self._val = new_value


c = C()
c.prop = "New value"
```

**Comment by adamtheturtle** [comment]
Even simpler, without `property`, just a simple descriptor:

```python
class D:
    def __get__(self, o: object, ot: type | None = None) -> None:
        return

    def __set__(self, o: object, v: int) -> None:
        return


class C:
    a = D()


c = C()
print(c.a)
c.a = 1
```

**Comment by adamtheturtle** [comment]
Fixed in #31
